src/test/java/de/tum/in/www1/artemis/util/DatabaseUtilService.java

```diff
@@ -516,6 +516,41 @@ public Exam addExam(Course course) {
         return exam;
     }
 
+    public Exam addExamToCourse(Course course) {
```

Comment:
What is the benefit of defining this here? (In comparison to the addExam() one above using ModelFactory.generateExam(course))

Reply:
This was actually unused code as the exam generation was refactored in another branch which was merged into develop. Good catch! I removed it